src/main/java/com/fasterxml/jackson/perf/json/JsonArbitraryFieldNameBenchmark.java

```diff
@@ -0,0 +1,151 @@
+package com.fasterxml.jackson.perf.json;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.BenchmarkMode;
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.annotations.OutputTimeUnit;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.concurrent.ThreadLocalRandom;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Supplier;
+
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.MICROSECONDS)
+public class JsonArbitraryFieldNameBenchmark {
+
+    public enum FactoryMode {
+        DEFAULT() {
+            @Override
+            JsonFactory apply(JsonFactory factory) {
+                return factory;
+            }
+        },
+        NO_INTERN() {
+            @Override
+            JsonFactory apply(JsonFactory factory) {
+                return factory.disable(JsonFactory.Feature.INTERN_FIELD_NAMES);
```

Comment:
These are problematic wrt Jackson 3.0 as method doesn't exist. Could it be changed to use JsonFactory.builder()? (otherwise can't merge to `master`)?

Reply:
Good call, I'll update to use JsonMapper/JsonFactry.Builder types